src/Calendar.jsx

```diff
@@ -208,6 +203,23 @@ class Calendar extends React.Component {
     });
   }
 
+  onBlur = (event) => {
+    const dateInput = DateInput.getInstance();
+    const rootInstance = this.rootInstance;
```

Comment:
Seems it's safe to move these into `setTimeout`.

Reply:
Done